src/api/config/options.yml.example

```diff
@@ -183,6 +183,14 @@ default: &default
   #    persistent: true
   #    passive: true
 
+  # For storing rails metrics in influxdb
+  # https://github.com/influxdata/influxdb-rails
```

Comment:
For admins this url doesn't really have much value as it does not explain any of the settings.